src/client/sandbox/node/index.js

```diff
@@ -37,6 +39,19 @@ export default class NodeSandbox extends SandboxBase {
         }
     }
 
+    _parseDocumentCharset () {
+        var metaCharset = NativeMethods.querySelector.call(document, '.' + CONST.SHADOW_UI_CHARSET_CLASSNAME);
+
+        if (metaCharset) {
+            metaCharset.parentNode.removeChild(metaCharset);
```

Comment:
As In understand it's covered by shadow UI, so I don't see the reason why we can't keep it.